test/api/test-font.c

```diff
@@ -119,6 +119,21 @@ test_face_createfortables (void)
   g_assert (freed);
 }
 
+static void
+test_font_empty_glyph_extents ()
+{
+  hb_face_t *face = hb_test_open_font_file ("fonts/empty-glyph.ttf");
+  hb_font_t *font = hb_font_create (face);
+  hb_face_destroy (face);
+  hb_glyph_extents_t extents;
+  g_assert (hb_font_get_glyph_extents (font, 1, &extents));
+  g_assert_cmpint (extents.x_bearing, ==, 0);
+  g_assert_cmpint (extents.y_bearing, ==, 0);
+  g_assert_cmpint (extents.width, ==, 0);
+  g_assert_cmpint (extents.height, ==, 0);
```

Comment:
Manually checked that hits the touched part however was more interesting if was non zero one, hard to reach part of the code anyway.

Reply:
What was the result before this change?

Reply:
Was also zero that's why the test isn't that good, apparently fonttools itself sets xMin xMax yMin yMax so not sure how to set those properly in a test font.

Reply:
Sorry, I don’t understand what this change is doing then to make any informed review.

Reply:
You are absolutely right, do you know some I can tweak xMin yMin xMax yMax without having to deal with

```
<!-- The xMin, yMin, xMax and yMax values
         will be recalculated by the compiler. -->
```

of fonttools. Some binary tweak maybe? Thanks

Reply:
Okay, apparently I don't understand this fully also, let me close so I can came with a better understanding and real test. Thanks